src/main/java/org/traccar/model/ExtendedModel.java

```diff
@@ -90,7 +90,7 @@ public void add(Map.Entry<String, Object> entry) {
     }
 
     public String getString(String key, String defaultValue) {
-        if (attributes.containsKey(key)) {
+        if (attributes.containsKey(key) && attributes.get(key) != null) {
             return attributes.get(key).toString();
```

Comment:
Null is a valid value. We can probably do something like this:

```suggestion
        if (attributes.containsKey(key)) {
            Object value = attributes.get(key);
            return value != null ? value.toString() : value;
```

Reply:
I though that would be the same as getString(key) would call getString(key, defaultValue: null).
So the question is when calling getString(key, someValue) and the actual key is defined and its value is null, do we return null or do we return someValue (the default value) ?
Both are reasonable to me, We just need to avoid Exceptions that cause unexpected behaviors.